### ISSUE TITLE
A great THANK YOU for the good times

### ISSUE THREAD

**Issue by aacoles** [5digits/dactyl#220]
I never contributed or was active in this project, but was an avid user for many years. 

At a certain point it was inevitable to have to move to FF57, and so I ended up figuring out how to replace most of the functionality with Saka Key and wiriting my own Greasemonkey scripts for the Pentadactyl plugins I was using.

Pentadactyl was an incredible extension and I want to deeply thank everyone who worked on it. Sad to see the decline of programmer-users and the user hostility (no unsigned extensions, removal of filesystem access for extensions, the removal of important extension APIs with no WebExtension equivalent), but not much we can do about it.

Thanks for the good times while they lasted!
A

**Comment by dnnr** [comment]
I second that wholeheartedly. Thanks for everything!

**Comment by glacambre** [comment]
Yup, a huge thank you to all the devs and contributors, Pentadactyl will be missed!
If Saka-key isn't close enough to Pentadactyl for you, you should check out Vim-Vixen and Tridactyl. They provide quite similar default keybindings and even have a command line (though nowhere as powerful Pentadactyl's).

**Comment by schmitmd** [comment]
Another voice to add to the appreciation.  This piece of software has been absolutely amazing throughout the years.  I would claim that it's been as indispensable, useful, and efficient for me as Vim itself, openssh, and git.  Never let me down, the documentation is excellent, and it definitely saved my bacon a bunch of times with that glorious tab completion.  I can't even count the number of times I kinda-sorta remembered a buzzword from 5 months ago and Pentadactyl was immediately like "oh, you mean _this_ random, obscure website or forum post?  No worries, I got you."   Thank you, thank you, and thank you.

**Comment by troyp** [comment]
Keep in mind, Pentadactyl still works fine on [Waterfox](https://www.waterfoxproject.org/) and [Pale Moon](https://www.palemoon.org/). There are currently a few minor issues on [Basilisk](https://www.basilisk-browser.org/), but they should be easy to fix (and wouldn't affect most users anyway).

Waterfox and Basilisk even support webextensions as well as XPCOM/XUL, so you install new FF extensions going forward.

There's nothing we can do about the decline of Mozilla, but we can support a fork. The more people who use them, the more chance they'll survive in the long term.

**Comment by torappinfo** [comment]
Pentadactyl for palemoon 28 lost some functionality.
1. plugins do not work.
2. "sanitize" does not work.

**Comment by smckown** [comment]
Sorry to post to such an old thread, but I want to express my thanks as well.  I'm a software developer that uses vim heavily, but was quite late in coming to Pentadactyl.  I love it.  One of its best features, aside from the 'normal' vim-style navigation controls, is the unite-like mechanism for selecting current buffers, or URLs from history, etc.

I'm still using Pentadactyl on Pale Moon 28.1.0, and it works fine for me.
